src/main/resources/datical/datical_deploy.bat.ftl

```diff
@@ -10,4 +10,18 @@
 
 -->
 <#include "/datical/datical_generic.ftl">
+<#include "/datical/datical_credentials.bat.ftl">
+${ddb_audit_user}
+${ddb_audit_pass}
+${ddb_user}
+${ddb_pass}
+<#--
```

Comment:
Can those lines be removed? Why are they commented out? Looks like a leftover from testing the changes.